corehq/apps/app_manager/tests/test_suite_remote_request.py

```diff
@@ -328,3 +328,21 @@ def test_prompt_itemset(self):
             suite,
             "./remote-request[1]/instance[@id='states']",
         )
+
+    def test_prompt_default_value(self, *args):
+        """Setting the default to "default_value"
+        """
+        self.module.search_config.properties[0].default_value = 'foo'
```

Comment:
The tiniest of nitpicks: if this were either `'foo'` or `3` it would be a valid expression which would be a nice touch.